index.html

```diff
@@ -13,53 +13,48 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <nav class="desktop-nav">
+    <div class="social-media">
+      <ul class="links">
+        <li><i class="fa fa-facebook"></i></li>
+        <li><i class="fa fa-twitter"></i></li>
+        <li>Logout</li>
+        <li>My page</li>
+        <li>korean</li>
+      </ul>
+    </div>
+    <header class="desktop-nav">
```

Comment:
Great design @SAZZAD1Q2 🤝

- [ ] [Optional] Kindly ensure your navbars are properly aligned just like in the given design.
- Your design
![your-nav](https://user-images.githubusercontent.com/63917725/227723439-2961e0fa-df2f-416b-b368-a073b2534412.PNG)
- Given design
![given-nav](https://user-images.githubusercontent.com/63917725/227723458-1d58586d-7fcc-48a9-bf41-9fb5bc2ac71f.PNG)